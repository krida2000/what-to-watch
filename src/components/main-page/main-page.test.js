import React from "react";
import renderer from "react-test-renderer";
import MainPage from "./main-page.jsx";

it(`Main page screen correctly render`, () => {
  const tree = renderer
      .create(
          <MainPage movies={[`test`]}/>).toJSON();

  expect(tree).toMatchSnapshot();
}
);
