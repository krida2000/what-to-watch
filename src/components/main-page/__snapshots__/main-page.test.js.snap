// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main page screen correctly render 1`] = `
<section
  className="film"
>
  <div>
    <img
      alt="Movie image"
      src="./img/avatar.jpg"
    />
    <p
      className="movie-title"
    >
      test
    </p>
  </div>
</section>
`;
